index.html

```diff
@@ -51,6 +53,238 @@ <h3>LET’S CONNECT</h3>
+                <div class="btn-text">See Project</div>
+              </div>
+            </div>
+          </div>
+        </div>
+        <!-- End of second card -->
+        <div class="card-works">
+          <img
+            src="./images/work/Tonic2.png"
+            alt="Tonic Image"
+            class="snapshot-portfolio"
+          />
+          <div class="left-block">
+            <div class="title-box">
+              <div class="project-title">Tonic</div>
+              <div class="canopy">
+                <div class="client">CANOPY</div>
+                <div class="counter"></div>
+                <div class="role">Back End Dev</div>
+                <div class="counter"></div>
+                <div class="year">2015</div>
+              </div>
+            </div>
+            <div class="description-box">
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required.
+            </div>
+            <div class="tag-box">
+              <ul class="tags">
+                <li class="tag">HTML</li>
+                <li class="tag">CSS</li>
+                <li class="tag">Javascript</li>
+              </ul>
+            </div>
+            <div class="action-box">
+              <div class="button">
+                <div class="btn-text">See Project</div>
+              </div>
+            </div>
+          </div>
+        </div>
+        <!-- End of third card -->
+        <div class="card-works">
+          <img
+            src="./images/work/multipost2.png"
+            alt="Tonic Image"
+            class="snapshot-portfolio"
+          />
+          <div class="left-block">
+            <div class="title-box">
+              <div class="project-title">Multi-post Stories</div>
+              <div class="canopy">
+                <div class="client">CANOPY</div>
+                <div class="counter"></div>
+                <div class="role">Back End Dev</div>
+                <div class="counter"></div>
+                <div class="year">2015</div>
+              </div>
+            </div>
+            <div class="description-box">
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required.
+            </div>
+            <div class="tag-box">
+              <ul class="tags">
+                <li class="tag">HTML</li>
+                <li class="tag">CSS</li>
+                <li class="tag">Javascript</li>
+              </ul>
+            </div>
+            <div class="action-box">
+              <div class="button">
+                <div class="btn-text">See Project</div>
+              </div>
+            </div>
+          </div>
+        </div>
+        <!-- End of fourth card -->
+      </section>
+      <!-- End of Work Section -->
+
+      <!-- ABOUT MYSELF SECTION -->
+      <section id="aboutmyself">
+        <div class="top">
+          <div class="h2">About <br />Myself</div>
```

Comment:
According to the project requirements, this content should be wrapped inside a `hx` tag.

<img width="349" alt="Screen Shot 2022-05-27 at 17 40 07" src="https://user-images.githubusercontent.com/82426682/170797256-5cfa9904-5df8-49d7-b977-91d63e612a01.png">

Kindly fix this.